src/Yoti.Auth/DigitalIdentity/DigitalIdentityService.cs

```diff
@@ -1,16 +1,27 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Net.Http;
 using System.Text;
 using System.Threading.Tasks;
+using Google.Protobuf;
 using Newtonsoft.Json;
 using Org.BouncyCastle.Crypto;
+using Yoti.Auth.Attribute;
 using Yoti.Auth.Exceptions;
+using Yoti.Auth.Profile;
 using Yoti.Auth.Web;
+using Yoti.Auth.ProtoBuf.Attribute;
+using Yoti.Auth.Share;
 
 namespace Yoti.Auth.DigitalIdentity
 {
     public static class DigitalIdentityService
     {
+        private const string identitySessionReceiptRetrieval = "/v2/receipts/{0}";
+        private const string identitySessionReceiptKeyRetrieval = "/v2/wrapped-item-keys/{0}";
+        private const string digitalIdentitySessionCreationEndpoint = "/v2/sessions";
```

Comment:
yes sure but in case in future i have set variable for future feature that can be added so we can use common variables. But i have simplified var names.

Reply:
> But i have simplified var names.
Did u push?